src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -160,7 +166,11 @@ const SavedSearchAlertEditForm: React.FC<SavedSearchAlertEditFormProps> = ({
               <Input
                 title="Alert Name"
                 name="name"
-                placeholder={entity.placeholder}
+                placeholder={
+                  isFallbackToGeneartedAlertNamesEnabled
```

Comment:
```suggestion
                  isFallbackToGeneratedAlertNamesEnabled
```

Reply:
Ahh, good catch. Fix one typo and introduce another! 😅